resources/aws_ec2.go

```diff
@@ -173,6 +173,7 @@ type AwsEc2Res struct {
 // chanStruct defines the type for a channel used to pass events and errors to watch.
 type chanStruct struct {
 	event awsEc2Event
+	state string
```

Comment:
Added this, because, now that we have instance state consts to use, we no longer need the awsEc2Event type. The next patch will refactor CheckApply which will include switching to the ec2 pkg consts. Once I do the same for snsWatch, we can get rid of the iota consts and the event field.